dist/schema-form.js

```diff
@@ -184,15 +184,15 @@ angular.module('schemaForm').provider('sfBuilder', ['sfPathProvider', function(s
 
         var children = args.fieldFrag.children || args.fieldFrag.childNodes;
```

Comment:
The [contribution docs](https://github.com/json-schema-form/angular-schema-form/blob/development/CONTRIBUTING.md) state that PRs should not include changes to `dist`